subprojects/wlroots.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://gitlab.freedesktop.org/wlroots/wlroots.git
-revision = 0.16
+revision = 5fb0007e0249388792f3772c30bfabf8d551dec0
```

Comment:
We'll want to change this to `0.17` now that it's released

Reply:
Done :smile:

Reply:
That was fast :)

Reply:
I'd come to the same conclusion and guess I was doing it as you posted the message :smile: